packages/bpk-component-card-list/README.md

```diff
@@ -0,0 +1,37 @@
+# bpk-component-card-list
+
+> Backpack card list component.
+
+## Installation
+
+Check the main [Readme](https://github.com/skyscanner/backpack#usage) for a complete installation guide.
+
+## Usage
+
+### BpkCardList
+
+```js
+import { BpkCardList } from '@skyscanner/backpack-web/bpk-component-card-list';
+
+export default () => (
+  <BpkCardList
+    title="Title"
+    description="Description"
+    buttonText="Button"
+    buttonHref="https://www.skyscanner.net"
+    onButtonClick={() => {}}
+  />
+);
+```
+
+## Props
+
+### BpkCardList
```

Comment:
You don't need this anymore, you can use instead TSDocs to autogenerate docs in Storybook